nlp_primitives/required_data.txt

```diff
@@ -1 +1 @@
-nltk-data.tar.gz
\ No newline at end of file
+nltk_data.tar.gz
```

Comment:
@rwedge @thehomebrewnerd do we actually need this?

Reply:
Would have to study a bit more to know for sure, but it looks like we are including the data we need by including the `data/nltk_data` directory, so it doesn't seem like that file is needed for anything.

Reply:
Removed